src/components/QuestionPlay/index.tsx

```diff
@@ -74,8 +79,8 @@ export default function QuestionPlay({
             Question {selectedQuestion.questionNum}
           </Title>
           {negativePointsMultiplier === 0 ? (
-            <Badge color="green" variant="filled">
-              Points: {points}
+            <Badge color={getBadgeColor(points, minQuestionPoint, maxQuestionPoint)} variant="filled">
```

Comment:
Not doing this when negative points?
Maybe we still do it for negative points but the negative point style can be outlined in red or some other color apart from these 5 colors?

Reply:
made the badge outlined for negative points